src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,102 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type PaginationProps = {
+  total: number;
+  perPage: string;
```

Comment:
The `perPage` prop is currently typed as a string, but it should be a number. Update the type of `perPage` in `PaginationProps` to `number` to ensure type consistency with its usage in the component.